includes/constants.php

```diff
@@ -3,13 +3,22 @@
 namespace Automattic\WP\Cron_Control;
 
 /**
- * Job queues
+ * Adjustable queue size and concurrency limits, to facilitate scaling
+ */
+$job_queue_size = defined( 'CRON_CONTROL_JOB_QUEUE_SIZE' ) && is_numeric( \CRON_CONTROL_JOB_QUEUE_SIZE ) ? absint( \CRON_CONTROL_JOB_QUEUE_SIZE ) : 10;
```

Comment:
Do we need max/min protection for these constants? (e.g. what happens when someone accidentally sets the queue size to 0? Or 1000?)